consensus/debugging.go

```diff
@@ -94,6 +36,14 @@ func (m *consensusModule) resetToGenesis(_ *messaging.DebugMessage) error {
 	if err := m.GetBus().GetPersistenceModule().Start(); err != nil { // reload genesis state
 		return err
 	}
+
```

Comment:
Just wondering why this now happens after the debug message is handled, also the `// Reset Persistence` is still before that.